integration/tests/basic/test_nonce.py

```diff
@@ -34,6 +35,7 @@ def test_get_receipt_sequence(self):
 
         self.check_transaction_list(tx_hash_list)
 
+    @pytest.mark.mainnet
```

Comment:
These tests fails very often on devnet , not shure about mainnet
@gigimon what do you think?

Reply:
Yes, I remember now. These tests are not stable. I guess you're right we should exclude it from run. @gigimon Are you ok with it?

Reply:
no, we don't need.

Reply:
removed